### ISSUE TITLE
Tuning

### ISSUE THREAD

**Issue by vince62s** [ModernMT/MMT#217]
Hello,
In the "mainstream" moses, tuning is not presented as "optional". On the contrary, it's pretty clear that the default weights will not work properly.

I am not clear why you say it's an "optional" step in the tuto.

Also, is the devset automatically heldout at corpus preparation ? (ie I don't have to specify a devset path at tuning time ?)

Grazie mille.

**Comment by cattoni** [comment]
Hi @vince62s 

In MMT tuning is reported as optional because the default weights assigned to a newly created engine (as the last step of the "mmt create" command) are not uniform as in moses: they come from past experience on other systems.
So they are considered in some way as "reasonable" to begin...
Clearly a tuning step on domain data is expected to improve the system, sometimes a lot.

Concerning the second question, yes, default of "mmt create" is to separate two small portions on the training set as dev and test sets, so the user can issue "mmt tune" and "mmt evaluate" without any additional info.
This default behaviour can be inhibited with the option --no-split in "mmt create": in this case the full training set is used to create the engine, but the user has to provide the dev or test set with the --path option when invocating "mmt tune" or "mmt evaluate".
